File: check-db.py
import sqlite3
import os

db_path = os.path.join(os.path.dirname(__file__), 'projects.db')

print(f'📊 Checking database: {db_path}\n')

try:
    conn = sqlite3.connect(db_path)
    cursor = conn.cursor()
    
    # Get all tables
    cursor.execute("SELECT name FROM sqlite_master WHERE type='table'")
    tables = cursor.fetchall()
    
    print('Tables in database:')
    for table in tables:
        print(f'  - {table[0]}')
        
        # Get row count
        cursor.execute(f'SELECT COUNT(*) FROM {table[0]}')
        count = cursor.fetchone()[0]
        print(f'    Rows: {count}')
        
        # Show first few rows
        if count > 0:
            cursor.execute(f'SELECT * FROM {table[0]} LIMIT 3')
            rows = cursor.fetchall()
            cursor.execute(f'PRAGMA table_info({table[0]})')
            columns = [col[1] for col in cursor.fetchall()]
            print(f'    Columns: {", ".join(columns)}')
    
    conn.close()
    
except Exception as e:
    print(f'❌ Error: {e}')

File: check-db.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('./database.db');

db.serialize(() => {
  console.log('📊 Database Status:\n');
  
  db.get('SELECT COUNT(*) as count FROM projects', (err, row) => {
    if (err) {
      console.log('❌ Error:', err.message);
      return;
    }
    console.log(`Projects: ${row.count}`);
  });
  
  db.get('SELECT COUNT(*) as count FROM tasks', (err, row) => {
    if (err) {
      console.log('❌ Error:', err.message);
      return;
    }
    console.log(`Tasks: ${row.count}`);
  });
  
  db.all('SELECT id, name, status, archived FROM projects', (err, rows) => {
    if (err) {
      console.log('❌ Error:', err.message);
      return;
    }
    console.log('\n📦 Projects:');
    rows.forEach(p => {
      const archived = p.archived ? '📦' : '✅';
      console.log(`  ${archived} ${p.id}. ${p.name} (${p.status})`);
    });
    
    db.close();
  });
});
